android/app/src/main/cpp/peer_audio_manager.cpp

```
@@ -1,287 +1,386 @@
+                     std::chrono::duration_cast<std::chrono::milliseconds>(drift)
+                         .count()));
+            nextTick = std::chrono::steady_clock::now();
         }
     }
 
     LOGI("Mixer tick loop ended");
 }
 
-void PeerAudioManager::sendAudioToPeer(const std::string& macAddress, const uint8_t* opusData, int opusSize, uint32_t seq) {
-    if (!jvm || !callbackObject) {
+void PeerAudioManager::sendAudioToPeer(const std::string& macAddress,
```

Clang (14.0.6)

[warning] method 'sendAudioToPeer' can be made static

(readability-convert-member-functions-to-static)